src/main/resources/application.properties

```diff
@@ -1,9 +1,19 @@
 #For groupmates: this should definitely be in gitignore in production or when connecting to a remotely hosted db.
 #This is only to make things easier for you.
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
-spring.datasource.username=postgres
-spring.datasource.password=mysecretpassword
-spring.jpa.hibernate.ddl-auto=create-drop
-spring.jpa.show-sql=true
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
-spring.jpa.properties.hibernate.format_sql=true
+#spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```

Comment:
Please don't change this file on origin. See the comment above

Reply:
I know I added my own application properties for my database, but I did comment them out so yes you can see them but it wouldn't pull anything.  I guess I was being lazy about it, I can copy paste them next time and un-comment the original.  I wasn't really trying to figure out gitignore at this moment either.

Reply:
just don't `git add` the file and the commit won't alter the file